packages/language-server-ruby/src/util/spawn.ts

```diff
@@ -6,7 +6,7 @@ import log from 'loglevel';
 
 export type SpawnOpts = {
 	stdin?: Observable<string>;
-	encoding?: string;
+	encoding?: BufferEncoding;
```

Comment:
I was unable to compile without this change.  Buffer.toString() on L53 was not able to ensure the type could be converted since `string` here could be anything.